docs/path-resolution.rst

```diff
@@ -38,6 +38,8 @@ compilation fails.
 
 The command-line compiler provides the *Host Filesystem Loader* - a rudimentary callback
 that interprets a source unit name as a path in the local filesystem.
+The default callback can be disabled using the ``--no-import-callback`` command-line option, in which
+case, an error will be thrown when attempting to load sources that have not been explicitly provided.
```

Comment:
How about this? The paragraph above already says that with no callback compilations fails.

```suggestion
By default, the command-line compiler provides the *Host Filesystem Loader* - a rudimentary callback
that interprets a source unit name as a path in the local filesystem.
This callback can be disabled using the ``--no-import-callback`` command-line option.
```